src/components/InstanceCard/InstanceCard.tsx

```diff
@@ -23,61 +26,79 @@ import { ScanAndDeliverBlurb } from "../EditionCard/ScanAndDeliverBlurb";
 export const InstanceCard: React.FC<{
   edition: WorkEdition;
   instance: Instance;
+  isFeaturedEdition?: boolean;
 }> = (props) => {
   // cookies defaults to be undefined if not fonud
   const [cookies] = useCookies([NYPL_SESSION_ID]);
 
   const edition = props.edition;
   const instance: Instance = props.instance;
+  const isFeaturedEdition = props.isFeaturedEdition;
   const previewItem = EditionCardUtils.getPreviewItem(instance.items);
   const isPhysicalEdition = EditionCardUtils.isPhysicalEdition(previewItem);
 
   return (
-    <Card
-      imageProps={{
-        src: EditionCardUtils.getCover(edition.links),
-        size: "xsmall",
-        aspectRatio: "original",
-        alt: "Cover",
-      }}
-      layout="row"
-      isBordered
-      isAlignedRightActions
-      id={`card-${instance.instance_id}`}
-      p="s"
+    <Box
+      border="1px"
+      borderColor="ui.border.default"
+      padding="s"
+      paddingBottom="l"
+      paddingRight="l"
     >
-      <CardHeading
-        level="h3"
-        size="heading6"
-        sx={{ span: { fontSize: "18px" } }}
+      <Flex gap="xs" flexDirection={{ base: "column", md: "row" }}>
+        {isPhysicalEdition && <CardRequiredBadge />}
+        {isFeaturedEdition && <FeaturedEditionBadge />}
+      </Flex>
+      <Card
+        imageProps={{
+          src: EditionCardUtils.getCover(edition.links),
+          size: "xsmall",
+          aspectRatio: "original",
+          alt: "Cover",
+        }}
+        layout="row"
+        isAlignedRightActions
+        id={`card-${instance.instance_id}`}
+        paddingTop="m"
       >
-        <Flex alignItems="center">
-          <span>
-            {edition.publication_date
-              ? edition.publication_date
-              : "Edition Year Unknown"}
-          </span>
-          {isPhysicalEdition && <PhysicalEditionBadge />}
-        </Flex>
-      </CardHeading>
-      <CardContent>
-        <div>
-          {EditionCardUtils.getPublisherAndLocation(
-            instance.publication_place,
-            instance.publishers
+        <CardHeading
+          level="h3"
+          size="heading6"
+          sx={{ span: { fontSize: "18px" } }}
+        >
+          <Flex alignItems="center">
+            <span>
+              {edition.publication_date
+                ? edition.publication_date
+                : "Edition Year Unknown"}
+            </span>
+            {isPhysicalEdition && <PhysicalEditionBadge />}
+          </Flex>
+        </CardHeading>
+        <CardContent>
+          <div>
+            {EditionCardUtils.getPublisherAndLocation(
+              instance.publication_place,
+              instance.publishers
+            )}
+          </div>
+          <div>{EditionCardUtils.getWorldCatElem(instance)}</div>
```

Comment:
Let's refactor this into a React component instead of a function that returns JSX.